data/data/baremetal/main.tf

```diff
@@ -3,8 +3,9 @@ provider "libvirt" {
 }
 
 provider "ironic" {
-  url          = var.ironic_uri
+  url          = "http://${var.bootstrap_provisioning_ip}:6385/v1"
   microversion = "1.52"
+  timeout      = 1200
```

Comment:
I've had to bump this up to 1200, which I think is going to cause issues with timeouts.  My connection in RDU took about 15 minutes for the images to finish downloading and Ironic to become available. This may be why CI has been failing on the dev-scripts PR.

Reply:
I've actually set it to 3600, but this doesn't cause any timeout issue with the installer.  The time out only begins after terraform stands up the infrastructure for bootstrap and masters.